rabbitmq/src/test/groovy/io/micronaut/rabbitmq/annotation/MandatorySpec.groovy

```diff
@@ -39,13 +40,15 @@ class MandatorySpec extends AbstractRabbitMQTest {
 
     void "test mandatory annotation applied to methods and method parameters"() {
         startContext()
+        MandatorySender1 mandatorySender1 = applicationContext.getBean(MandatorySender1)
         MandatorySender3 mandatorySender3 = applicationContext.getBean(MandatorySender3)
         ParametricSender parametricSender = applicationContext.getBean(ParametricSender)
         ChannelPoolListener listener = applicationContext.getBean(ChannelPoolListener)
 
         when:
         mandatorySender3.send("this message will be returned")
         parametricSender.send(false, "this message will NOT be returned")
+        mandatorySender1.sendNotMandatory("this message will not be returned either")
```

Comment:
are we testing somehow that this message is not returned? 

Should we add something like `listener.returns.stream().noneMatch(it -> it.body == 'this message will not be returned either'.bytes)` ?

Reply:
Sure, we can do that 👍 https://github.com/micronaut-projects/micronaut-rabbitmq/pull/508/commits/d3c266ba09b4d3fe7645b952478eda91fc5b2f01